.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+
+# The bigtable-dpe team is the default owner for anything not
+# explicitly taken by someone else.
+*                               @GoogleCloudPlatform/bigtable-dpe
```

Comment:
As far as I know, we can't grant access to teams in a different GitHub organization.

Reply:
Is that because of auth issues or because of our policy?

I can make a googleapis@ version of the teams either way, but if it's for policy I worry about creating two different teams and needed to check them for access / removing access.

Reply:
Made the correct team and adjusted the owners file.